### PR TITLE
feat(preset): add corporate preset - INNO-917

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

- `yarn dist` now generate css/js for all presets (not all preset use js)
- Add "corporate" preset containing only header and footer (and all dependencies)

How to test the corporate preset:
- Get or generate the css and js for corporate preset
- Use file `test-preset.html`attached to https://webgate.ec.europa.eu/CITnet/jira/browse/INNO-917
- Update path to europa-corporate.css and europa-corporate.js in the file
- display the html file; header and footer should be correctly displayed and funcitonal

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

* [ ] `package.json` is up-to-date and `@ec-europa/ecl-base` is part of the dependencies
* [ ] I have checked the dependencies
* [ ] I have given the fractal status “ready” to my component
* [ ] I have declared `@define mycomponent` in the SCSS file
* [ ] I have specified `margin: 0;` on the CSS component
* [ ] I have provided tests
* [ ] I follow the naming guidelines
* [ ] the component supports composition
* [ ] there are no hardcoded strings (all content come from the context)
* [ ] I have filled the README.md file (at least a few lines)
* [ ] My component is listed in the root README
* [ ] My PR has the right label(s)

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 4b9f6d3a1c1c7b16b818bfd2fc455062de10174d

https://deploy-preview-559--europa-component-library.netlify.com

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 3ba3cea844498832a7c216658ef49f9965f44c1b

https://deploy-preview-559--europa-component-library.netlify.com

File: packages/presets/ecl-preset-corporate/_imports.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
+// Base
+@import '~@ec-europa/ecl-preset-base';
```
Comment (yhuard) [inline packages/presets/ecl-preset-corporate/_imports.scss]: should be `@import '~@ec-europa/ecl-preset-base/imports';`

Comment (yhuard) [reply packages/presets/ecl-preset-corporate/_imports.scss]: this remark is also valid for the `_imports.scss` file in the `ecl-preset-full`

**Review by yhuard — CHANGES_REQUESTED** [review]
2 small changes requested

**Review by yhuard — APPROVED** [review]

**Pull request merged**